cdc/sink/flowcontrol/flow_control.go

```diff
@@ -82,13 +97,7 @@ func (c *TableFlowController) Consume(commitTs uint64, size uint64, blockCallBac
 		}
 	}
 
-	c.queueMu.Lock()
-	defer c.queueMu.Unlock()
-	c.queueMu.queue.PushBack(&commitTsSizeEntry{
-		commitTs: commitTs,
-		size:     size,
-	})
-
+	c.enqueueSingleMsg(msg, commitTs, size)
```

Comment:
this PR doesn't split the big transaction , just make a mark ?

Reply:
This is divided into two steps：
1. make sink support split transactions, so that a single large transaction can be written concurrently to downstream, which can significantly reduce synchronization latency
2. implement batch resolve mechanism, which can alleviate the oom problem to a certain extent